src/actions/index.js

```diff
@@ -168,7 +168,7 @@ export function fetchProjects(parms, stateKey) {
 
 function tagSwipeFriendly(workflows) {
   return map((workflow) => {
-    workflow.swipe_verified = !!workflow.configuration.swipe_enabled && isValidSwipeWorkflow(workflow)
+    workflow.swipe_verified = workflow.mobile_friendly && isValidSwipeWorkflow(workflow)
```

Comment:
I think there still needs to be a check here for `workflow.configuration.swipe_enabled`. It can either be here or in the `isValidSwipeWorkflow()` function. That is, just because a workflow is `mobile_friendly`doesn't necessarily mean it will be `swipe_enabled`. In the future, as the app expands in functionality, we may have a workflow that is `mobile_friendly` but not necessarily `swipe_enabled`, and just because a project meets all the conditions to be a swipe workflow doesn't mean the project owner would want it to be considered such.

Open to debate on this though.

Reply:
Isn't the whole point of this pr: https://github.com/zooniverse/Panoptes-Front-End/pull/3894 that a user can select for their workflow to be displayed in the app? I'm pretty sure we only set a workflow to `mobile_friendly` if a user makes a decision to have their workflow appear in the app.

So we aren't displaying all projects that meet the criteria, we are only displaying all of the apps that have met the criteria and have been selected to appear in the app. In that sense the attribute name `mobile_friendly` is misleading